firstvoices/backend/views/import_job_views.py

```diff
@@ -44,7 +53,47 @@
         ],
     ),
     create=extend_schema(
-        description=_("Add a batch import job."),
+        description=_(
+            "Queue a new batch import job and create a model instance to track the job. The first stage "
+            "of the import job workflow is to validate and execute a dry-run of the provided CSV, and "
+            "display any issues with the CSV or rows. It also displays how many rows will be imported,"
+            " ignored or contain any errors with their details. Once the dry-run has been successfully "
+            "completed, the confirm action can be used to do the database import for the provided CSV."
+        ),
+        responses={
+            201: OpenApiResponse(
+                description=doc_strings.success_201, response=ImportJobSerializer
+            ),
+            400: OpenApiResponse(description=doc_strings.error_400_validation),
+            403: OpenApiResponse(description=doc_strings.error_403),
+            404: OpenApiResponse(description=doc_strings.error_404_missing_site),
+        },
+        parameters=[
+            site_slug_parameter,
+            id_parameter,
+        ],
+    ),
+    destroy=extend_schema(
+        description="Deletes a single import-job and its associated file and result for the specified site. "
+        "This action does not delete any of the entries imported by the import-job.",
+        responses={
+            204: OpenApiResponse(description=doc_strings.success_204_deleted),
+            403: OpenApiResponse(description=doc_strings.error_403),
+            404: OpenApiResponse(description=doc_strings.error_404_missing_site),
+        },
+        parameters=[
+            site_slug_parameter,
+            id_parameter,
+        ],
+    ),
+    confirm=extend_schema(
+        description=_(
+            "This action proceeds to import the entries from the CSV file for the specified import-job. "
+            "It requires the CSV to be validated, and a successful dry-run for the specified import-job. "
+            "The response shall return with the current status for the import-job in the `status` field, "
+            "which is different from the `validationStatus` field. "
+            "*Note: No request body is required for this action.*"
```

Comment:
Again I think we could adjust this to focus more on using the API. Maybe something like:

"Starts importing the data, as described in the validationReport. In order to succeed, the validationStatus must already be COMPLETE and there must be no other import jobs in progress for this site. When finished, the status will be COMPLETE"